### PR TITLE
Add Reproduction Results for Pyserini Repo Steps

### PR DESCRIPTION
**Pull request by hrouzegar**
**Environment Details:**
-**Operating System:** Ubuntu 22.04.4 LTS
-**System RAM:** 24GB
 
**Experience:**
The process was smooth with no significant issues, except for the wget command for downloading the dataset not functioning as expected. I utilized the wget command from the Anserini documentation which worked without issue. The scripts ran flawlessly after setting up the environment correctly, and the step-by-step guidance was clear and easy to follow.

### CODE REVIEW

**Review by lintool — APPROVED** [review]

**Review by lintool — CHANGES_REQUESTED** [review]
Please fix conflicts? Sort entries chronologically.

File: docs/conceptual-framework.md [hunk 1]
```diff
@@ -335,3 +335,4 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
 + Results reproduced by [@emadahmed19](https://github.com/emadahmed19) on 2024-04-28 (commit [`9db2584`](https://github.com/castorini/pyserini/commit/9db25847829a656d1c9eacb267bf745f7522dd14))
 + Results reproduced by [@CheranMahalingam](https://github.com/CheranMahalingam) on 2024-05-05 (commit [`f817186`](https://github.com/castorini/pyserini/commit/f8171863df833ac02ff427d4823a1085e63094bf))
 + Results reproduced by [@billycz8](https://github.com/billycz8) on 2024-05-08 (commit [`c945c50`](https://github.com/castorini/pyserini/commit/c945c50c3e22e3c6ecae50a55aed48853617acc0))
++ Results reproduced by [@hrouzegar](https://github.com/hrouzegar) on 2024-05-11 (commit [`f6f0dd6`](https://github.com/castorini/pyserini/commit/f6f0dd6dc4cdbc77b9b6fac10ed21dd6bed76d13))
```
Comment (lintool) [inline docs/conceptual-framework.md:338]: This does not appear to be a valid commit? page 404s
https://github.com/castorini/pyserini/commit/f6f0dd6dc4cdbc77b9b6fac10ed21dd6bed76d13

Comment (hrouzegar) [reply docs/conceptual-framework.md]: Apologies for the confusion; I've updated the commit ID and fixed the link. Also, I've sent you an email regarding the PhD position for further discussion. Thank you!

**Comment by lintool** [comment]
https://github.com/castorini/anserini/discussions/2428

Please fix.

**Pull request closed without merging**